File: Exemplos M10/App_IntentDigitalHUB_Flutter/Elgin_IntentDigitalHUB_Flutter-font/lib/IntentDigitalHubService/BRIDGE/set_senha.dart
import 'package:intent_digital_hub/IntentDigitalHubService/BRIDGE/bridge_command.dart';

class SetSenha extends BridgeCommand {
  final String _senha;
  final bool _habilitada;

  SetSenha(this._senha, this._habilitada) : super('SetSenha');

  @override
  functionParameters() {
    return '"senha"'
        ':'
        '"$_senha"'
        ','
        '"habilitada"'
        ':'
        '$_habilitada';
  }
}

File: Exemplos M10/App_IntentDigitalHUB_Kotlin/Elgin_IntentDigitalHUB_Kotlin-font/app/src/main/java/com/elgin/intent_digital_hub/IntentDigitalHubService/BRIDGE/Commands/SetSenha.kt
package com.elgin.intent_digital_hub.IntentDigitalHubService.BRIDGE.Commands

import com.elgin.intent_digital_hub.IntentDigitalHubService.BRIDGE.BridgeCommand

class SetSenha(private val senha: String, private val habilitada: Boolean) :
    BridgeCommand("SetSenha") {
    override fun functionParameters(): String {
        return "\"senha\"" + ":" + "\"" + senha + "\"" + "," +
                "\"habilitada\"" + ":" + habilitada
    }
}